tests/pages/account.test.tsx

```diff
@@ -0,0 +1,228 @@
+          value: "test@test.test",
+        },
+      ],
+    };
+
+    const component = mount(
+      <ThemeProvider theme={lightTheme}>
+        <GlobalStyle />
+        <Layout>
+          <Account sortedIdentities={mockedSortedResponse} />
+        </Layout>
+      </ThemeProvider>,
+    );
+
+    const emailTrash = component.find({ type: IdentityTypes.EMAIL });
+    const phoneTrash = component.find({ type: IdentityTypes.PHONE });
+
+    expect(emailTrash).toHaveLength(0);
+    expect(phoneTrash).toHaveLength(2);
+  });
+
+  test("it should appear for both email and phone if phone and email list > 1", () => {
+    const mockedSortedResponse: SortedIdentities = {
+      phoneIdentities: [
+        {
+          id: "8f79dcc1-530b-4982-878d-33f0def6a7cf",
+          primary: false,
+          status: "validated",
+          type: ReceivedIdentityTypes.PHONE,
+          value: "0622116655",
+        },
+        {
+          id: "8f79dcc1-530b-4982-878d-33f0def6a7cf2",
+          primary: false,
+          status: "validated",
+          type: ReceivedIdentityTypes.PHONE,
+          value: "0622116652",
+        },
+      ],
+      emailIdentities: [
+        {
+          id: "8f79dcc1-530b-4982-878d-33f0def6a7cf",
+          primary: false,
+          status: "validated",
+          type: ReceivedIdentityTypes.EMAIL,
+          value: "test@test.test",
+        },
+        {
+          id: "8f79dcc1-530b-4982-878d-33f0def6a7cf2",
+          primary: false,
+          status: "validated",
+          type: ReceivedIdentityTypes.EMAIL,
+          value: "test2@test.test",
+        },
+      ],
+    };
+    const component = mount(
+      <ThemeProvider theme={lightTheme}>
+        <GlobalStyle />
+        <Layout>
+          <Account sortedIdentities={mockedSortedResponse} />
+        </Layout>
+      </ThemeProvider>,
+    );
+
+    const emailTrash = component.find({ type: IdentityTypes.EMAIL });
+    const phoneTrash = component.find({ type: IdentityTypes.PHONE });
+
+    expect(emailTrash).toHaveLength(2);
+    expect(phoneTrash).toHaveLength(2);
+  });
+
+  test("it should appear for neither email nor phone if phone and email list = 0 (a specific messsage should appear instead)", () => {
+    const mockedSortedResponse: SortedIdentities = {
+      phoneIdentities: [],
+      emailIdentities: [],
+    };
+    const component = mount(
+      <ThemeProvider theme={lightTheme}>
+        <GlobalStyle />
+        <Layout>
+          <Account sortedIdentities={mockedSortedResponse} />
+        </Layout>
+      </ThemeProvider>,
+    );
+
+    const emailTrash = component.find({ type: IdentityTypes.EMAIL });
+    const phoneTrash = component.find({ type: IdentityTypes.PHONE });
+    const noPhone = component.find(".no-phone");
+    const noEmail = component.find(".no-email");
+
+    expect(emailTrash).toHaveLength(0);
+    expect(phoneTrash).toHaveLength(0);
+    expect(noPhone).toHaveLength(2);
+    expect(noEmail).toHaveLength(2);
```

Comment:
Why 2? Shouldn't there be one per type?

Reply:
When there is on element displayed, Enzyme acts as if the length is 2

Reply:
🤔 what does it display if you console.log `noPhone`?

Reply:
the `console.log(noPhone);` returns `ReactWrapper {}`

Reply:
Maybe with https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/debug.html

Reply:
OK fixed aa72665 I decided to use `.hostNodes()` method to filter down the result of `.find()` to HTML elements only.

Besides I decided to check if `emailTrash` 's behaviour is as expected if there are 3 identities (because so far I was only testing with 0, 1 or 2)